app/kitchensink/index.html

```diff
@@ -912,7 +912,8 @@ <h2 id="Cards" class="page-header">Cards
                         <h3 class="mdl-card__title-text">CBP Super Bowl 50 Air and Marine Operations Security</h3>
                       </div>
                       <div class="mdl-card__supporting-text">
-                        Personnel with U.S. Customs and Border Protection conduct operations in advance of Super Bowl 50 in Santa Clara, Calif., 2016. (U.S. Customs and Border Protection Photo by Glenn Fawcett)
+                        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type
```

Comment:
Don't need to change this since it matches the picture and is an example

Reply:
Reverted back.